KPI-Schedule/Controller/ScheduleController.swift

```diff
@@ -1,23 +1,35 @@
-//
-//  ScheduleController.swift
-//  KPI-Schedule
-//
-//  Created by Dima Tavlui on 05.06.2023.
-//
-
 import UIKit
 
 class ScheduleController: UITableViewController {
+    
+    var scheduleManager = ScheduleManager()
+    var firstWeekSchedule = [Day]()
+    var secondWeekSchedule = [Day]()
+    var scheduleWeek = [Day]()
+    var defaults = UserDefaults()
+    var daysWithPairs = [Day]()
 
     override func viewDidLoad() {
         super.viewDidLoad()
 
-        // Uncomment the following line to preserve selection between presentations
-        // self.clearsSelectionOnViewWillAppear = false
 
-        // Uncomment the following line to display an Edit button in the navigation bar for this view controller.
-        // self.navigationItem.rightBarButtonItem = self.editButtonItem
     }
-
+    
+    @IBAction func changeWeek(_ sender: UISegmentedControl) {
+        switch sender.selectedSegmentIndex {
+        case 0:
+            scheduleWeek = firstWeekSchedule
+        case 1:
+            scheduleWeek = secondWeekSchedule
+        default:
+            scheduleWeek = firstWeekSchedule
+        }
+        DispatchQueue.main.async {
+            self.tableView.reloadData()
+        }
+    }
```

Comment:
Eliminated the default case in the switch statement because the default action was the same as the action in case 0
```suggestion
    @IBAction func changeWeek(_ sender: UISegmentedControl) {
    let selectedSchedule = sender.selectedSegmentIndex == 1 ? secondWeekSchedule : firstWeekSchedule
    scheduleWeek = selectedSchedule
    
    DispatchQueue.main.async {
        self.tableView.reloadData()
    }
}

```